packages/fiori/src/UploadCollection.ts

```diff
@@ -130,8 +124,10 @@ class UploadCollection extends UI5Element {
 	 * <ul>
 	 * <li><code>None</code></li>
 	 * <li><code>SingleSelect</code></li>
+	 * <li><code>SingleSelectBegin</code></li>
+	 * <li><code>SingleSelectEnd</code></li>
 	 * <li><code>MultiSelect</code></li>
-	 * <li><code>Delete</code></li>
+	 * <li><code>Delete</code> - this mode has no effect. The delete button is controlled by the <code>hideDeleteButton</code> property of UploadCollectionItem</li>
```

Comment:
I suggest describing this in the [[TOPIC] 2.0 breaking changes list ](https://github.com/SAP/ui5-webcomponents/issues/4460) to whatever you would ideally see as the API related to the delete btn and delete mode.

Reply:
This behavior is introduced with https://github.com/SAP/ui5-webcomponents/pull/3404 and described in  **Components API review** . I'm just fixing some things related to it and making the docs more clear.

Reply:
Yes I know, but we can plan to remove the mode if it does not have effect and purpose

<li><code>Delete</code> - this mode has no effect. The delete button is controlled by the <code>hideDeleteButton</code> property of UploadCollectionItem</li>

or remove the property and leave the Delete mode and so on.

Reply:
Good point. I have added it.